### PR TITLE
2017MC ZATo2LA01j_5f_NLO_FXFX and WAToLNuA01j_5f_NLO_FXFX with 306000…

### PR DESCRIPTION
**Pull request by junholeephy**
… central pdf

### CODE REVIEW

**Comment by junholeephy** [comment]
These are to be used for 2017 Dataset. 
Pdf setting has been changed from 292200 to 306000 "https://github.com/cms-sw/genproductions/pull/1455".

**Comment by efeyazgan** [comment]
- Please commit to master and follow the guidelines in https://twiki.cern.ch/twiki/bin/viewauth/CMS/Moriond18MC (also for pdf settings)

- Please move the cards to MadGraph5_aMCatNLO/cards/production/2017/13TeV/WAToLNuA01j_5f_NLO_FXFX/
and MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZATo2LA01j_5f_NLO_FXFX/ZATo2LA01j_5f_NLO_FXFX_proc_card.dat
And follow

**Comment by junholeephy** [comment]
Considering discrepancy of photon_eta (2nd page of [1]), I have put these on mg26x branch. Should I move it to the master branch?

[1] https://indico.cern.ch/event/663254/contributions/2710009/attachments/1518570/2371281/Report_ZG_20170905T.pdf

**Comment by efeyazgan** [comment]
OK. It can stay in mg26. Please do the other changes.

**Comment by junholeephy** [comment]
Since "gridpack_generation.sh" is differently written with MG26x and master branch, I think I am not able to follow twiki indicated settings, e.g. $DEFAULT_PDF_SETS = lhaid. $DEFAULT_PDF_MEMBERS = reweight_PDF. 
May I ask you any comment about this?

**Comment by efeyazgan** [comment]
@kdlong

**Comment by kdlong** [comment]
Yes unfortunately that's the case. Will have a fix by the end of the day.

**Comment by junholeephy** [comment]
So anything else for me to do in order to merge the PR?

**Comment by kdlong** [comment]
If you want to set $DEFAULT_PDF_SETS = lhaid, this will be supported. $DEFAULT_PDF_MEMBERS = reweight_PDF is no longer needed.

**Comment by kdlong** [comment]
You can try with https://github.com/cms-sw/genproductions/pull/1471

You should see a summary of the cross sections when you run ./runcmsgrid.sh after producing the gridpack. Please also check that the cross sections reported here look sensible.

**Comment by govoni** [comment]
yes I can

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/WAToLNuA01j_5f_NLO_FXFX/WAToLNuA01j_5f_NLO_FXFX_run_card.dat [hunk 1]
```diff
@@ -75,30 +77,38 @@
  F        = fixed_ren_scale  ! if .true. use fixed ren scale
  F        = fixed_fac_scale  ! if .true. use fixed fac scale
  91.188   = muR_ref_fixed    ! fixed ren reference scale 
- 91.188   = muF1_ref_fixed   ! fixed fact reference scale for pdf1
- 91.188   = muF2_ref_fixed   ! fixed fact reference scale for pdf2
+ 91.118   = muF_ref_fixed    ! fixed fact reference scale
+# 91.188   = muF1_ref_fixed   ! fixed fact reference scale for pdf1
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/WAToLNuA01j_5f_NLO_FXFX/WAToLNuA01j_5f_NLO_FXFX_run_card.dat]: should this be removed?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/WAToLNuA01j_5f_NLO_FXFX/WAToLNuA01j_5f_NLO_FXFX_run_card.dat [hunk 2]
```diff
@@ -75,30 +77,38 @@
...
 # Renormalization and factorization scales (advanced and NLO options)  *
 #***********************************************************************
- F        = fixed_QES_scale  ! if .true. use fixed Ellis-Sexton scale
- 91.188   = QES_ref_fixed    ! fixed Ellis-Sexton reference scale
- 1        = muR_over_ref     ! ratio of current muR over reference muR
- 1        = muF1_over_ref    ! ratio of current muF1 over reference muF1
- 1        = muF2_over_ref    ! ratio of current muF2 over reference muF2
- 1        = QES_over_ref     ! ratio of current QES over reference QES
+# F        = fixed_QES_scale  ! if .true. use fixed Ellis-Sexton scale
+# 91.188   = QES_ref_fixed    ! fixed Ellis-Sexton reference scale
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/WAToLNuA01j_5f_NLO_FXFX/WAToLNuA01j_5f_NLO_FXFX_run_card.dat]: should the commented lines be removed?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/WAToLNuA01j_5f_NLO_FXFX/WAToLNuA01j_5f_NLO_FXFX_run_card.dat [hunk 3]
```diff
@@ -75,30 +77,38 @@
...
 #***********************************************************************
- .true.   = reweight_scale   ! reweight to get scale dependence
-  0.5     = rw_Rscale_down   ! lower bound for ren scale variations
-  2.0     = rw_Rscale_up     ! upper bound for ren scale variations
-  0.5     = rw_Fscale_down   ! lower bound for fact scale variations
-  2.0     = rw_Fscale_up     ! upper bound for fact scale variations
+  1.0 = rw_rscale ! muR factors to be included by reweighting
+  1.0 = rw_fscale ! muF factors to be included by reweighting
+ True   = reweight_scale   ! reweight to get scale dependence
+#  0.5     = rw_Rscale_down   ! lower bound for ren scale variations
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/WAToLNuA01j_5f_NLO_FXFX/WAToLNuA01j_5f_NLO_FXFX_run_card.dat]: same here and everywhere, unless I am missing something

**Comment by perrozzi** [comment]
hi, can the commented lines be removed?

**Comment by perrozzi** [comment]
ping

**Pull request closed without merging**